WNPRC_EHR/resources/queries/study/wnprcFullHistory/Full History.qview.xml

```diff
@@ -19,7 +19,7 @@
        <sort column="date" descending="true"/>
    </sorts>
    <filters>
-       <filter column="dataset/name" operator="in" value="assignments;behavetrem;biopsy;blood;clinremarks;deaths;encounters;drug;housing;necropsy;problem;weight;tb;clinpathRuns;foodDeprives;ultrasounds;pregnancy_outcomes;breeding_encounters,chairing,restraint"/>
+       <filter column="datasetname" operator="in" value="assignments;behavetrem;biopsy;blood;clinremarks;deaths;encounters;drug;housing;necropsy;problem;weight;tb;clinpathRuns;foodDeprives;ultrasounds;pregnancy_outcomes;breeding_encounters;research_ultrasounds,chairing,restraint;watertotal"/>

```

Comment:
Should this be just 'dataset' - i don't see a 'datasetname' column in the columns listed above.

Reply:
We made a change to the wnprcFullHistory.sql which is here:
WNPRC_EHR/resources/queries/study/wnprcFullHistory.sql
And it should match datasetname